cypress/tests/core/basic/content.js

```diff
@@ -87,7 +87,7 @@ describe('Add Content Tests', () => {
     cy.findByLabelText('/my-page').children('td').eq(1).click();
 
     cy.get('.top-menu-menu .delete').click();
-    cy.get('.modal.active').contains('Potential link breakage');
+    cy.get('.modal.active').contains('View broken links list');
```

Comment:
Maybe we should reflect some of the other use-cases the new modal is able to handle in the tests.

Reply:
@pgrunewald I understand the idea of testing other cases, but would be of the opinion of keeping it concise to avoid slowing down the CI with extra content creation and a longer cypress test. I consider that the implementation of both backend and frontend is demonstrated in the current use-case, but let me know what do you think

Reply:
Sorry for the late response. I was mentioning this point, because judging from the changed files it would seem that only some text in the modal has been changed. But again, testing too fine-grained on subtle text parts might lead to flaky tests, which are burdensome to fix. LGTM in that regard.